tests/unit/test_api.py

```diff
@@ -144,6 +151,15 @@ def test_assign_trunk_port(self, m_run_task):
 
         m_run_task.assert_called_once()
 
+    def test_assign_trunk_port_stp_edge(self, m_run_task):
+        self.net_runr.conf_trunk_port(self.testhost,
+                                      self.testport,
+                                      self.testvlan,
+                                      trunked_vlans=self.testvlans,
+                                      stp_edge=True)
+
+        m_run_task.assert_called_once()
```

Comment:
I think this and the other should asset_called_once_with to make sure the vars are passed through